docs/modules/ROOT/partials/Intro_to_Mill_Header.adoc

```diff
@@ -1,26 +1,26 @@
+{mill-github-url}[Mill] is a {language} build tool that aims for performance,
+maintainability, and flexibility:
 
+* *Performance*: Mill's xref:Tasks.adoc[build graph] automatically caches and parallelizes build
+  steps (called "_targets_"), minimizing the amount of work that needs to be done, and
+  automatically profiling your build so you can optimize it. Even for large codebases with
+  hundreds of modules, Mill keeps your workflows fast and responsive.
 
-* **Mill builds are easy to extend by non-experts**: anyone can extend a Mill build
-with a few lines of code, without needing to learn a complicated plugin framework,
-allowing end users the freedom to fit their build logic to their own unique requirements
+* *Maintainability*: Mill build targets are self contained without side-effects, making it easy
+  to trace the data-flow of your build. Mill also provides better IDE support
+  (via xref:{language}_Installation_IDE_Support.adoc[IntelliJ or VSCode]) than any other build tool,
+  letting you use "_jump-to-definition_" to navigate around your build
+  as easily as any application codebase. This makes maintaining Mill builds
+  a far easier than the "_guess and grep_" workflows common with other build tools.
```

Comment:
The "better" might be arguable. Maybe, we just say "good" .

The "jump to definition" is most likely a unique selling point though.